dashboard/app/models/levels/level.rb

```diff
@@ -574,24 +574,25 @@ def clone_with_name(new_name)
   # @param [String] new_suffix The suffix to append to the name of the original
   #   level when choosing a name for the new level, replacing any existing
   #   name_suffix if one exists.
-  def clone_with_suffix(new_suffix)
+  def clone_with_suffix(new_suffix, editor_experiment: nil)
```

Comment:
Please update doc comments in this file with a note about the `editor_experiment` parameter.